src/web/testing.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>Unsigned Addons</title>
+  </head>
+  <body>
+    <ul>
+      <li>
+        <a
+          href="https://mozilla.github.io/FirefoxColor/build/web/firefox-color-dev-unsigned.xpi"
```

Comment:
Use a relative URL so that the links point to the xpis that are built with the current deployment.

E.g.

```html
<li><a href="firefox-color-dev-unsigned.xpi">Unsigned add-on for localhost:8080</a>
<li><a href="firefox-color-stage-unsigned.xpi">Unsigned add-on for color.stage.mozaws.net</a>
<li><a href="firefox-color-unsigned.xpi">Unsigned add-on for color.firefox.com</a>
```